app/libs/loaders/Leaf_autoloader.php

```diff
@@ -35,30 +35,29 @@
 //error_log(print_r($site_paths, true));
 $site_paths = $site_paths['data'][0];
 
-if (is_dir($curr_dir . '/libs/php-commons')) {
-    //Commenting this out for testing purposes, this will be uncommented before going to prod.
+/** Here down is old loader stuff, will be deprecated as we go along getting everything into a single source of code. */

```

Comment:
Good.  But can you put some comment brackets around the "old" code that will be deprecated?